README.md

```diff
@@ -18,7 +18,7 @@ Please refer to the [getting started guide](https://kudo.dev/docs/) documentatio
 
 ## Community Meetings
 
-We have open community meetings every 2nd and 4th Wednesday of the month at 9:00 a.m. PST. (17:00 UTC)
+We have open community meetings every 2nd and 4th Thursday of the month at 9:00 a.m. PST. (17:00 UTC)
```

Comment:
The meeting as is has been scheduled just recently and is reflected on the KUDO Calendar as _Every 2 weeks on Thursday_. This is not the same as 2nd and 4th Thursday; in May already, it'll be on the **first** Thursday.

Can we just keep the _every other week_ schedule and adjust the wording appropriately?

Reply:
ah.. that is correct.  thank you.  I'll update wording here.

Reply:
actually @meichstedt.. if we go with every other Thurs... this has a likelihood at some pt of colliding with other eco-system meetings.   I think I would like to NOT change the wording here.  For one, it is what a group of people opted into.  I will update the calendar invite tomorrow after providing a chance to change my thoughts on it

Reply:
@kensipe I’m basically fine with either as long as the wording matches the actual schedule and we don't run into conflicts with internal meetings.